From e971f343e1916615b3d1852bcebc88c9e2ba9ea0 Mon Sep 17 00:00:00 2001
From: Sabri Ünal <yakushabb@gmail.com>
Date: Thu, 21 May 2026 18:43:49 +0300
Subject: [PATCH] Fix build errors

Signed-off-by: Sabri Ünal <yakushabb@gmail.com>
---
 CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 158ad29..74b9ef4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,11 +11,6 @@ set(OS_BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
 set(CMAKE_PREFIX_PATH Sources/Externals)
 
-if(POLICY CMP0054)
-	# Escape variables in if
-	cmake_policy(SET CMP0054 OLD)
-endif()
-
 include(cmake/FindSDL2.cmake)
 if(NOT SDL2_FOUND)
 	message(FATAL_ERROR "SDL 2.0 not found, set ENV{SDL2DIR} to point to SDL 2.0, and optionally set the cmake var SDL2_LIBRARY_TEMP to the lib dir")
--
libgit2 1.7.2

